talentmap_api/fsbid/urls/cycle_job_categories.py

```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url
+from rest_framework import routers
+
+from talentmap_api.fsbid.views import cycle_job_categories as views
+
+router = routers.SimpleRouter()
+
+urlpatterns = [
+    url(r'^$', views.FSBidCycleCategoriesView.as_view(), name="cycle-categories"),
+    url(r'^job_categories/$', views.FSBidCycleJobCategoriesView.as_view(), name="cycle-job-categories"),
+    url(r'^job_categories/status/$', views.FSBidCycleJobCategoriesStatusesView.as_view(), name="cycle-job-categories-statuses"),
```

Comment:
```suggestion
    url(r'^job_categories/statuses/$', views.FSBidCycleJobCategoriesStatusesView.as_view(), name="cycle-job-categories-statuses"),
```
? it's plural everywhere else - up to you